File: CodeSignal/Edge of the Ocean/go - 5:5/matrixElementsSum.go
func matrixElementsSum(matrix [][]int) int {
    sum := 0
    for row := 0; row < len(matrix[0]); row++ {
        for col := 0; col < len(matrix); col++ {
            if matrix[col][row] == 0 {
                break;
            }
            sum += matrix[col][row]
        }
    }
    return sum
}

File: CodeSignal/Edge of the Ocean/py - 5:5/matrixElementsSum.py
def matrixElementsSum(matrix):
    thesum, bannedcols = 0, set()
    for i in range(len(matrix)):
        for j in range(len(matrix[i])):
            if matrix[i][j] == 0:
                bannedcols.add(j)
            if j not in bannedcols:
                thesum += matrix[i][j]
    return thesum